etc/kayobe/environments/ci-multinode/cephadm.yml

```diff
@@ -35,9 +36,17 @@ cephadm_pools:
   - name: vms
     application: rbd
     state: present
+cephadm_pools_manila_cephfs_native:
+  - name: cephfs_data
+    application: cephfs
+    state: present
```

Comment:
Could use the state to make this conditional.

Reply:
Done